privacy.html

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Privacy Policy
 ---
-
+<script>function gaOptout() {document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';window[disableStr] = true;}</script>

```

Comment:
Should this maybe open a message window (alert()) to say that the cookie has been set? ATM, nothing happens if the user clicks the link, which is a bit unexpected.